synapse/_scripts/update_synapse_database.py

```diff
@@ -97,8 +97,8 @@ def main() -> None:
     # Load, process and sanity-check the config.
     hs_config = yaml.safe_load(args.database_config)
 
-    if "database" not in hs_config:
-        sys.stderr.write("The configuration file must have a 'database' section.\n")
+    if "database" not in hs_config and "databases" not in hs_config:
+        sys.stderr.write("The configuration file must have a 'database' or 'databases' section.\n")
         sys.exit(4)
```

Comment:
I think we do similar (and more thorough) error checking when we parse the config anyway... I wonder if we can just get rid of these lines of code?

https://github.com/matrix-org/synapse/blob/493c2fc44abcf3457953cc2f6f23509ff7855253/synapse/config/database.py#L97-L123

Reply:
I wasn't clear what this check was for. it does seem like there is another check lower down. Should I remove it?

Reply:
I went ahead and removed it

Reply:
Yeah, I think it is fine to remove it. The error message seems pretty much the same too.